tests/integration/local/invoke/invoke_integ_base.py

```diff
@@ -75,3 +78,35 @@ def get_command_list(
             command_list = command_list + ["--region", region]
 
         return command_list
+
+    def get_build_command_list(
+        self,
+        template_path=None,
+        cached=None,
+        parallel=None,
+        use_container=None,
+    ):
+        command_list = [self.cmd, "build"]
+
+        if template_path:
+            command_list = command_list + ["-t", template_path]
```

Comment:
[minor] I would use `append` instead.

Reply:
Right, but I wanted to keep the way it is been used for other commands.